<<<<<<< HEAD
<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>donation</class>
 <widget class="QDialog" name="donation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1292</width>
    <height>749</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1400</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{

font-family: century gothic;

	background-color: rgb(255, 255, 255)


}





</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>411</width>
     <height>631</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(196, 136, 136);
font family: century-gothic;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color: rgba(0,0,0,0);
border-bottom: 2px solid rgba(0,0,0,230);
padding-bottom:7px;</string>
    </property>
    <property name="placeholderText">
     <string>Full Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color: rgba(0,0,0,0);
border-bottom: 2px solid rgba(0,0,0,230);
padding-bottom:7px;</string>
    </property>
    <property name="placeholderText">
     <string>Enter Age</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color: rgba(0,0,0,0);
border-bottom: 2px solid rgba(0,0,0,230);
padding-bottom:7px;</string>
    </property>
    <property name="placeholderText">
     <string>Blood Group</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>350</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color: rgba(0,0,0,0);
border-bottom: 2px solid rgba(0,0,0,230);
padding-bottom:7px;</string>
    </property>
    <property name="placeholderText">
     <string>Are you taking medications? y/n</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>290</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color: rgba(0,0,0,0);
border-bottom: 2px solid rgba(0,0,0,230);
padding-bottom:7px;</string>
    </property>
    <property name="placeholderText">
     <string>Enter Weight</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_7">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>410</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color: rgba(0,0,0,0);
border-bottom: 2px solid rgba(0,0,0,230);
padding-bottom:7px;</string>
    </property>
    <property name="placeholderText">
     <string>Any health issue?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="donate">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>480</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#donate{

 	background-color: rgb(202, 103, 103);
border-radius: 5px;

}
QPushButton#donate:hover{
	background-color: rgb(224, 226, 255);

}

QPushButton#donate:pressed{

padding-left:5px;
padding-top:5px;

}
</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:40px;
background-color: rgb(0, 0, 0);
color:rgb(235, 203, 203)</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/donate.webp</normaloff>images/donate.webp</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/images/Blue, white and green Medical care logo.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>1201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>century gothic</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 0, 0);
color:rgb(231, 166, 166)</string>
   </property>
   <property name="text">
    <string>                                                                        D  O N A T E    B L O O D</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>1220</x>
     <y>80</y>
     <width>63</width>
     <height>681</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 0, 0);
color:rgb(231, 166, 166)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>230</y>
     <width>501</width>
     <height>511</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/images/ic1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
=======
<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>donation</class>
 <widget class="QDialog" name="donation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1292</width>
    <height>749</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1400</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{

font-family: century gothic;

	background-color: rgb(255, 255, 255)


}





</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>411</width>
     <height>631</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(196, 136, 136);
font family: century-gothic;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color: rgba(0,0,0,0);
border-bottom: 2px solid rgba(0,0,0,230);
padding-bottom:7px;</string>
    </property>
    <property name="placeholderText">
     <string>Full Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color: rgba(0,0,0,0);
border-bottom: 2px solid rgba(0,0,0,230);
padding-bottom:7px;</string>
    </property>
    <property name="placeholderText">
     <string>Enter Age</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color: rgba(0,0,0,0);
border-bottom: 2px solid rgba(0,0,0,230);
padding-bottom:7px;</string>
    </property>
    <property name="placeholderText">
     <string>Blood Group</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>350</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color: rgba(0,0,0,0);
border-bottom: 2px solid rgba(0,0,0,230);
padding-bottom:7px;</string>
    </property>
    <property name="placeholderText">
     <string>Are you taking medications? y/n</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>290</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color: rgba(0,0,0,0);
border-bottom: 2px solid rgba(0,0,0,230);
padding-bottom:7px;</string>
    </property>
    <property name="placeholderText">
     <string>Enter Weight</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_7">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>410</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color: rgba(0,0,0,0);
border-bottom: 2px solid rgba(0,0,0,230);
padding-bottom:7px;</string>
    </property>
    <property name="placeholderText">
     <string>Any health issue?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="donate">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>480</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#donate{

 	background-color: rgb(202, 103, 103);
border-radius: 5px;

}
QPushButton#donate:hover{
	background-color: rgb(224, 226, 255);

}

QPushButton#donate:pressed{

padding-left:5px;
padding-top:5px;

}
</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:40px;
background-color: rgb(0, 0, 0);
color:rgb(235, 203, 203)</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/donate.webp</normaloff>images/donate.webp</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/images/Blue, white and green Medical care logo.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>1201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>century gothic</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 0, 0);
color:rgb(231, 166, 166)</string>
   </property>
   <property name="text">
    <string>                                                                        D  O N A T E    B L O O D</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>1220</x>
     <y>80</y>
     <width>63</width>
     <height>681</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 0, 0);
color:rgb(231, 166, 166)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>230</y>
     <width>501</width>
     <height>511</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/images/ic1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
>>>>>>> 2b3e0b8 (first commit)
